class.jetpack.php

```diff
@@ -1559,6 +1559,19 @@ public static function is_development_mode() {
 		return apply_filters( 'jetpack_development_mode', $development_mode );
 	}
 
+	/**
+	 * Whether the site is currently onboarding or not.
+	 * A site is considered as being onboarded if it currently has an onboarding token.
+	 *
+	 * @access public
+	 * @static
+	 *
+	 * @return bool True the site is currently onboarding, false otherwise
```

Comment:
Should be "True if the site..."

Reply:
Oops. Fixed in https://github.com/Automattic/jetpack/pull/8432/commits/0bded7e89bb734296080ff9bc74eb163f7186cb8.